var/spack/repos/builtin/packages/neurodamus-model/package.py

```diff
@@ -6,6 +6,26 @@
 import os
 
 
+_REBUILD_NEURODAMUS_TPL = """#!/bin/sh
+set -e
+if [ "$#" -eq 0 ]; then
+    echo "******* Neurodamus builder *******"
+    echo "Syntax:"
+    echo "$0 <mods_dir> [add_include_flags] [add_link_flags]"
+    echo
+    echo "NOTE: mods_dir is literally passed to nrnivmodl. If you only have the mechanism mods"
+    echo " and wish to build neurodamus you need to include the neurodamus-specific mods."
+    echo " Under \$NEURODAMUS_ROOT/share you'll find the whole set of original mod files, as"
+    echo " well as the neurodamus-specific mods alone. You may copy/link them into your directory."
+    exit 1
+fi
+
+# run with nrnivmodl in path
+set -x
+'{nrnivmodl}' -incflags '{incflags} '"$2" -loadflags '{loadflags} '"$3" "$1"
+"""
+
```

Comment:
It's bit ugly to have this at the top of package. May be one of the below would be better?
* have a method with `@after_install` decorator
* have this script along with `package.py` and it will be copied during / after install